ext/session/session.stub.php

```diff
@@ -43,35 +43,45 @@ function session_commit(): bool {}
  * @param callable|object $open
  * @param callable|bool $close
  */
-function session_set_save_handler($open, $close = UNKNOWN, callable $read = UNKNOWN, callable $write = UNKNOWN, callable $destroy = UNKNOWN, callable $gc = UNKNOWN, callable $create_sid = UNKNOWN, callable $validate_sid = UNKNOWN, callable $update_timestamp = UNKNOWN): bool {}
+function session_set_save_handler(
+    $open,
+    $close = UNKNOWN,
+    callable $read = UNKNOWN,
+    callable $write = UNKNOWN,
+    callable $destroy = UNKNOWN,
+    callable $gc = UNKNOWN,
+    callable $create_session_id = UNKNOWN,
+    callable $validate_session_id = UNKNOWN,
+    callable $update_timestamp = UNKNOWN
+): bool {}
 
 function session_cache_limiter(?string $cache_limiter = null): string|false {}
 
 function session_cache_expire(?int $new_cache_expire = null): int|false {}
 
-function session_set_cookie_params(array|int $lifetime_or_options, ?string $path = null, ?string $domain  = null, ?bool $secure = null, ?bool $httponly = null): bool {}
+function session_set_cookie_params(array|int $lifetime_or_options, ?string $path = null, ?string $domain = null, ?bool $secure = null, ?bool $httponly = null): bool {}
 
 function session_start(array $options = []): bool {}
 
 interface SessionHandlerInterface
 {
     /** @return bool */
-    public function open(string $save_path, string $session_name);
+    public function open(string $path, string $name);
 
     /** @return bool */
     public function close();
 
     /** @return string */
-    public function read(string $key);
+    public function read(string $id);
 
     /** @return bool */
-    public function write(string $key, string $val);
+    public function write(string $id, string $value);
```

Comment:
```suggestion
    public function write(string $id, string $data);
```
Possibly, as you use $data in session_decode.

Reply:
Good idea! We could also use `$content` maybe (although I prefer data here), as asked in https://github.com/php/php-src/pull/6238#discussion_r497010679. Can we then standardize on using `$data` as a param name for similar params? :)

Reply:
Yes, I think `$data` is better than `$content`. I think you need to change `updateTimestamp()` to use it as well. At least based on docs, the argument there seems to have the same meaning.